easybuild/easyconfigs/j/Java/Java-13.0.2.eb

```diff
@@ -0,0 +1,14 @@
+name = 'Java'
+version = '13.0.2'
+
+homepage = 'https://openjdk.java.net'
+description = """Java Platform, Standard Edition (Java SE) lets you develop and deploy
+ Java applications on desktops and servers."""
+
+toolchain = SYSTEM
+
+source_urls = ['https://download.java.net/java/GA/jdk%(version)s/d4173c853231432d94f001e99d882ca7/8/GPL/']
```

Comment:
The download URL does not work for me, I'm always prompted to accept Oracle's license. Is that not the case for you?
```suggestion
# Downloads are subject to license.
# Available from https://www.oracle.com/java/technologies/javase-jdk13-downloads.html
```

Reply:
I was mistaken, current `source_urls` do work to download OpenJDK. I was trying to make it work with Oracle's JDK which is not the intention. Sorry for the noise.